app/Console/Commands/MigrateFromSqliteToMysql.php

```
@@ -0,0 +1,191 @@
+        // Cached config means .env is never loaded, so the operator's MySQL/MariaDB settings would be
+        // ignored here and the DB_CONNECTION rewrite below would not take effect — refuse until cleared.
+        if (App::configurationIsCached()) {
+            $this->components->error('Configuration is cached, so your MySQL/MariaDB settings in .env are not loaded.');
+            $this->line("  Run 'php artisan config:clear' first, then re-run this command.");
+
+            return self::FAILURE;
+        }
+
+        $sqlitePath = storage_path('database.sqlite');
+
+        if (! File::exists($sqlitePath) || File::size($sqlitePath) === 0) {
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Illuminate\Support\Facades\File' in method 'handle'. (undefined)

(StaticAccess)